pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', 'build/x86_64_xenial' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'run', '-v', '/workspace:/workspace', '-w', '/workspace/src/',  'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', '/bin/bash', '-c', 'make purge && make -j4 DISTRO=xenial deb' ]
```

Comment:
You could use `-j` instead of `-j4` to let `make` pick the level of parallelism...

Reply:
Not done?..

Reply:
Somehow my other comment with a ton of detail got lost, guess I won't use "edit" anymore, I can't use `-j`, as it causes the build to run out of memory. Leaving it as-is.